tests/tests/helpers.rs

```diff
@@ -108,6 +100,48 @@ pub async fn websocket_server_with_subscription() -> (SocketAddr, WsServerHandle
 		})
 		.unwrap();
 
+	module
+		.register_subscription("subscribe_5_ints", "n", "unsubscribe_5_ints", |_, pending, _| {
```

Comment:
Now we can call `pipe_from_stream` more than once, should we also have a test to make sure that we can?